src/api/app/views/webui2/webui/request/_change_devel_request_dialog.html.haml

```diff
@@ -0,0 +1,38 @@
+.modal-dialog{ role: 'document' }
+  .modal-content
+    = form_tag({ action: :change_devel_request, project: @project, package: @package }, method: :post) do
+      .modal-header
+        %h5.modal-title Change Devel Project Request
+      .modal-body
+        - if @current_devel_project && @current_devel_package
+          %p
+            Do you want to request to change the devel project for #{package_link @package} from
+            \#{project_or_package_link project: @current_devel_project}?
+        - else
+          %p
+            Do you want to request to set the devel project for #{package_link @package}?
+        .form-group
+          - if @current_devel_project && @current_devel_package
+            = label_tag('devel-project', 'New Devel project (leave free to delete the current one):')
+          - else
+            = label_tag('devel-project', 'Devel project:')
+          = text_field_tag('devel-project', '', required: true, class: 'form-control')
+        .form-group
+          = label_tag(:description, 'Description:')
+          = text_area_tag(:description, '', row: 3, class: 'form-control')
+      .modal-footer
+        = render partial: 'webui2/shared/dialog_action_buttons'
+
+:javascript
+  $("#devel-project").autocomplete({
```

Comment:
This should be a in Javascript file. It's also using the same code as in `src/api/app/assets/javascripts/webui2/request.js` (beside a few more parameters, but this could be taken into account in the new function)